CHANGELOG.md

```diff
@@ -179,7 +206,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 
-- If user does not have marketing data (organization and cost center IDs) set in their orderForm at time of checkout, set it
+- If user does not have marketing data (organization and cost center IDs) set in their orderForm at time of checkout,
+  set it
```

Comment:
Were these break lines added automatically? The standard is for these changelog entries to each be in their own lines, without breaks, AFAIK. It might not be a problem, but I thought it was weird that this whole file was changed to add these breaks.

Reply:
Sorry, it was my IDE.